components/playlist/media_detector_component_installer.cc

```diff
@@ -139,7 +154,10 @@ void RegisterMediaDetectorComponent(
   if (!cus)
     return;
 
-  NOTIMPLEMENTED();
+  auto installer = base::MakeRefCounted<component_updater::ComponentInstaller>(
+      std::make_unique<MediaDetectorComponentInstallerPolicy>(
```

Comment:
As we only use one component for playlist, `MediaDetectorComponentInstallerPolicy` can use below constansts directly instead of passing as args?

Reply:
Yup. Narrowed scope of constants.